examples/js-avo-inspector-example/src/App.js

```diff
@@ -8,6 +8,7 @@ const inspector = new Inspector.AvoInspector({
   env: Inspector.AvoInspectorEnv.Dev,
   version: "1.0.0",
   appName: "Demo App",
+  suffix: "1"
```

Comment:
Is this an optional parameter? Should we label that somehow in the examples?

Reply:
added comments